code/modules/cargo/bounty.dm

```diff
@@ -158,46 +161,43 @@
 
 // Called lazily at startup to populate bounties_list with random bounties.
 /datum/controller/subsystem/cargo/proc/setupBounties()
-	for(var/i = 0; i < 3; ++i)
+
+	for(var/i = 0; i < 6; ++i)
 		CHECK_TICK
 		var/subtype = pick(subtypesof(/datum/bounty/item/assistant))
 		try_add_bounty(new subtype)
 
-	for(var/i = 0; i < 1; ++i)
+	for(var/i = 0; i < 4; ++i)
```

Comment:
Leave these values as they are.
If more bounties are wanted, then the "additional bounties"-bounty can be used to get them.
It might be worth to tweak that so it can be used repeatetly.